packages/neos-ui-editors/src/Editors/Range/index.js

```diff
@@ -42,7 +42,7 @@ class RangeEditor extends PureComponent {
         const {options} = this.props;
         const {target} = event;
 
-        let value = parseInt(target.value, 10);
+        let value = parseFloat(target.value, 10);
```

Comment:
i guess we should use `parseFloat` based on the currently selected type for the property?

The `Editor` props currently dont contain the type, but id be fine with adding it to the props.

Alternatively i have fetched the type in a custom plugin like this

(pseudo code)

```ts
const nodeType = useNodeType(
    useCurrentlyFocusedNode()!.nodeType
)!;
const propertyType = nodeType.properties?.[props.identifier]?.type;
```

Reply:
I'm currently working on a easier solution: Something like `const useParseInt = options.step % 1 === 0;`